funcs/chirps/chirps_crop_func.py

```diff
@@ -0,0 +1,85 @@
+"""
+This script downloads CHIRPS global daily dataset and register with dcat
+"""
+import os
+import urllib
+import datetime
+import netCDF4
+import xarray
+
+from dtran.argtype import ArgType
+from dtran.ifunc import IFunc, IFuncType
+
+
+class CHIRPSCropFunc(IFunc):
+    id = "chirps_crop_func"
+    description = """
+    An adapter that downloads CHIRPS dataset, crop by given time and spatial constraint.
+    """
+    func_type = IFuncType.CROPPING_TRANS
+    friendly_name: str = "CHIRPS crop func"
+    inputs = {
+        "start_date": ArgType.String,
+        "end_date": ArgType.String,
+        "lat_min": ArgType.Number(optional=True),
```

Comment:
use the same argument names as the cropping function instead of lat/long min/max.